tests/testthat/test-model-summary.R

```diff
@@ -204,8 +204,3 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
     })
   })
 }) # closing withr::with_options
-
```

Comment:
I don't :), but even then I prefer to avoid white space changes like this because it's cosmetic churn that increases the chance of a conflict.  However, in this case, it's really just the end-result diff that's misleading because this branch had all the bbr.bayes code that was removed (i.e. there's not just one commit where I said "let's remove these trailing new lines").  Here are the commits that come into play, including merges from main that added new tests:

  * a7bfd08a feat: added model_summary.bbi_stan_model that wraps read_fit_model()
  * 1db55dde Merge branch 'develop' into bbr_stan_alpha
  * f71282c8 Merge branch 'main' into bbr_stan_alpha
  * 76645549 Merge branch 'main' into bbr_stan_alpha
  * 00bf9515 tree-wide: prune bbr.bayes-side bits

Reply:
Or, to say that another way: `main` introduced these white space additions and it conflicted with new tests that were in this branch at the time (before the bbr.bayes split).  When resolving the conflict, I unsurprisingly did not keep the large number of spaces before the new tests.  And then when later removing these tests (for the bbr.bayes split) I didn't add back the same number of trailing spaces (which would only be useful for minimizing the `main...bbr-bayes-split` diff).